src/components/helpers/dom_helpers.ts

```diff
@@ -259,3 +259,50 @@ function isOtherMobileOS() {
 export function isMobileOS() {
   return isAndroid() || isIOS() || isOtherMobileOS();
 }
+
+/** Return a base64 representation of the current spreadsheet, with the grid canvas and all the figures */
+export function getSpreadsheetAsBase64() {
```

Comment:
So ... when we open a pivot in fullscreen we use a dialog-like style with a backdrop. But we cannot have two separate viewports, so we convert the spreadsheet to a png, and use that as background image 🙃